### DISCUSSION TITLE
MacVim Snapshot 172

### DISCUSSION THREAD

**Discussion by ychin** [macvim-dev/macvim#1210] [Releases]
Updated to Vim 8.2.3455.

Apologies for the late update, as I have not had much time to work on MacVim. This update is primarily to catch up to latest Vim. Future MacVim releases in near future will go back to more bug fixes / features, and addressing potential issues caused by macOS 12.

Compatibility
====================

Requires macOS 10.9 or above.

Script interfaces have compatibility with these versions:

- Lua 5.4
- Perl 5.18
- Python2 2.7
- Python3 3.9
- Ruby 3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/macvim-dev/macvim/releases/tag/snapshot-172'>MacVim Snapshot 172</a>.</em>

**Comment by dlam26** [comment]
dunno much, but from Snapshot 171 -> 172 I'm getting an odd warning message when running `/Applications/MacVim.app/Contents/bin/mvim` from terminal  (mac)

```
Error detected while processing /Users/dlam/.vim/plugin/fugitive.vim:
line  527: E1208: -complete used without allowing argumentsline  528: E1208: -complete used without allowing argumentsline  532: E1208: -complete used without allowing argumentsline  536: E1208: -complete used without allowing argumentsPress ENTER or type command to continue
```

...and I guess it's complaining about these lines in me fugitive.vim  e.g.  https://github.com/tpope/vim-fugitive/blob/master/plugin/fugitive.vim

```
527 exe 'command! -bar -bang -nargs=0 -complete=customlist,fugitive#CompleteObject GRemove exe fugitive#RemoveCommand(<line1>, <count>, +"<range>", <bang>0, "<mods>", <q-args>, [<f-args>])'
528 exe 'command! -bar -bang -nargs=0 -complete=customlist,fugitive#CompleteObject GDelete exe fugitive#DeleteCommand(<line1>, <count>, +"<range>", <bang>0, "<mods>", <q-args>, [<f-args>])'
529 exe 'command! -bar -bang -nargs=1 -complete=customlist,fugitive#CompleteObject GMove   exe fugitive#MoveCommand(  <line1>, <count>, +"<range>", <bang>0, "<mods>", <q-args>, [<f-args>])'
530 exe 'command! -bar -bang -nargs=1 -complete=customlist,fugitive#RenameComplete GRename exe fugitive#RenameCommand(<line1>, <count>, +"<range>", <bang>0, "<mods>", <q-args>, [<f-args>])'
531 if exists(':Gremove') != 2 && get(g:, 'fugitive_legacy_commands', 1)
532   exe 'command! -bar -bang -nargs=0 -complete=customlist,fugitive#CompleteObject Gremove exe fugitive#RemoveCommand(<line1>, <count>, +"<range>", <bang>0, "<mods>", <q-args>, [<f-args>])'
533         \ '|echohl WarningMSG|echo ":Gremove is deprecated in favor of :GRemove"|echohl NONE'
534 endif
535 if exists(':Gdelete') != 2 && get(g:, 'fugitive_legacy_commands', 1)
536   exe 'command! -bar -bang -nargs=0 -complete=customlist,fugitive#CompleteObject Gdelete exe fugitive#DeleteCommand(<line1>, <count>, +"<range>", <bang>0, "<mods>", <q-args>, [<f-args>])'
537         \ '|echohl WarningMSG|echo ":Gdelete is deprecated in favor of :GDelete"|echohl NONE'
538 endif
```

**Reply by eirnym** [reply]
Figutive doesn't play well with recent version of Vim in general. I had this in Arch Linux. I suppose you should fill an issue in plugin repo rather than here

**Reply by ychin** [reply]
Did you have the most up-to-date version of fugitive @dlam26 ?

**Comment by gyuhyun-lee** [comment]
Anyone having an issue on MacOS Monterey? I'm getting just a blank screen, and re-installing doesnt seem to work.

**Reply by ychin** [reply]
I don't have Monterey installed but was planning on installing it this weekend since it seems to be relatively stable. I'll report back.
